demisto_sdk/commands/common/hook_validations/readme.py

```diff
@@ -43,6 +55,7 @@ def __init__(self, file_path: str, ignored_errors=None, print_as_warnings=False,
         self.file_path = Path(file_path)
         self.pack_path = self.file_path.parent
         self.node_modules_path = self.content_path / Path('node_modules')
+        self.file_text = (open(file_path)).read()
```

Comment:
let's talk about it, i'm a bit concerned with opening it once for all validations, without being able to know when(and if) it is closed.